src/sui/sui_window.c

```diff
@@ -690,6 +698,22 @@ static gboolean CTRL_J_K_on_press(GtkAccelGroup *group, GObject *obj,
     return TRUE;
 }
 
+static gboolean CTRL_Q_on_press(GtkAccelGroup *group, GObject *obj,
+        guint keyval, GdkModifierType mod, gpointer user_data){
+    SuiWindow *self;
+
+    if (mod != GDK_CONTROL_MASK) return FALSE;
+    if (keyval != GDK_KEY_q){
+        ERR_FR("unknown keyval %d", keyval);
+        return FALSE;
+    }
+
+    self = SUI_WINDOW(user_data);
+    gtk_widget_destroy(GTK_WIDGET(self));
```

Comment:
It would be better to call `sui_application_get_instance` + `sui_application_exit`.